components/Hero.tsx

```diff
@@ -14,25 +16,51 @@ export default function Hero() {
   const [account] = useAtom(accountAtom)
   const currentChapter = useAtomValue(currentChapterAtom)
 
+  const [imageMedia, setImageMedia] = useState('/assets/images/main-image.jpg')
+
+  const updateMedia = () => {
+    let result = '/assets/images/main-image.jpg'
+
+    if (typeof window !== 'undefined') {
+      const aspectRatio = window.innerWidth / window.innerHeight
+
+      if (aspectRatio < 0.6) {
+        result = '/assets/images/main-image-mobile-tall.jpg'
+      } else if (aspectRatio < 1) {
+        result = '/assets/images/main-image-mobile.jpg'
+      } else if (aspectRatio < 1.4) {
+        result = '/assets/images/main-image-tablet.jpg'
+      } else if (aspectRatio >= 2) {
+        result = '/assets/images/main-image-wide.jpg'
+      }
+    }
+
+    setImageMedia(result)
+  }
+
+  useEffect(() => {
+    updateMedia()
+    window.addEventListener('resize', updateMedia)
+    return () => window.removeEventListener('resize', updateMedia)
+  })
+
   return (
     <div className="flex h-full grow items-end justify-center">
       <Image
-        src="/assets/images/main-image.jpg"
+        src={imageMedia}
         alt="Saving Satoshi"
         fill
         quality={100}
         loading="eager"
         priority
         className="-z-20 h-full w-full object-cover"
+        objectPosition="50% 34%"
       />
```

Comment:
I did some exploration with this and It seems like If we want to have the responsive image selection to choose different images as opposed to simply selecting different quality we might want to opt for a `<picture>` element like so
```
      <picture className="absolute -z-20 h-full w-full object-cover hero-image-position">
        <source srcSet="/assets/images/main-image-mobile-tall.jpg" media="(max-aspect-ratio: 3/5)"/>
        <source srcSet="/assets/images/main-image-mobile.jpg" media="(max-aspect-ratio: 1)"/>
        <source srcSet="/assets/images/main-image-tablet.jpg" media="(max-aspect-ratio: 13/10)"/>
        <source srcSet="/assets/images/main-image-wide.jpg" media="(min-aspect-ratio: 2)"/>
        <img
          loading="eager"
          src="/assets/images/main-image.jpg"
          alt="Saving Satoshi Landing image"
          className="absolute -z-20 h-full w-full object-cover hero-image-position"
        />
      </picture>
```